Buildings/Experimental/DHC/Plants/BaseClasses/PartialPlant.mo

```diff
@@ -1,10 +1,10 @@
 within Buildings.Experimental.DHC.Plants.BaseClasses;
 partial model PartialPlant
-  "Partial class for modeling a central plant"
+  "Partial class for modeling a plant"
   replaceable package Medium=Buildings.Media.Water
     constrainedby Modelica.Media.Interfaces.PartialMedium
     "Service side medium";
-  replaceable package MediumHea_b=Buildings.Media.Water
+  replaceable package MediumHea_b=Buildings.Media.Steam
```

Comment:
I'll defer to your judgment on this, but wouldn't water be the appropriate default for the heating supply medium? I guess it depends on which is more common, and I don't really know (and my quick google search didn't yield good evidence for either).